scripts/mirror.js

```diff
@@ -212,33 +212,53 @@ const copyStatement = (data) => {
 /**
  * @param {SupportStatement} originalData
  * @param {SupportStatement} sourceData
+ * @param {string} destination
+ * @param {string} source
+ * @param {Array.<RegExp, string>} notesRepl
  * @returns {SupportStatement}
  */
-const bumpChromeAndroid = (originalData, sourceData) => {
+const bumpGeneric = (
+  originalData,
+  sourceData,
+  destination,
+  source,
```

Comment:
All that sounds great to me!  I'm working on cleaning up this script bit by bit to prepare for build-time mirroring.  This script hasn't gotten too much attention since it was first written... :P